content/applications/inventory_and_mrp/inventory/shipping/setup/sendcloud_shipping.rst

```diff
@@ -0,0 +1,227 @@
+==========================================
+Set up Sendcloud shipping services in Odoo
+==========================================
+
+Sendcloud is a shipping service aggregator. Sendcloud facilitates the integration of European
+shipping carriers with Odoo. Once integrated, the user can select the shipping carrier on inventory
+operations in their Odoo database.
+
+.. note::
+   Here is `Sendcloud's documentation <https://support.sendcloud.com/hc/en-us/articles
+   /360059470491-Odoo-integration>`_ on setting up the integration.
```

Comment:
This would serve better as a `seealso` tag instead of a `note`
```suggestion
.. seealso::
   Here is `Sendcloud's documentation <https://support.sendcloud.com/hc/en-us/articles
   /360059470491-Odoo-integration>`_ on setting up the integration.
```